src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +71,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating(){
+        return true;
```

Comment:
It seems that this method will return true regardless of the type of the Command. With this implementation, you should probably be returning the member variable `this.isMutating`.

Reply:
Another method of accomplishing this LO that you might like to explore is to have the child classes override the isMutating() method.